developer/installation/installation-windows.md

```diff
@@ -0,0 +1,124 @@
+# Installation for Windows
+
+## Install prerequisites:
+
+**Note: These commands all assume you are running them from an administrator shell**
+
+### Install node
+Download and run the installer from the [NodeJs site](https://nodejs.org)
+
+### Install Chocolately
+
+Visit the [Chocolatey site](https://chocolatey.org/install) and follow the instructions
+
+
+### Install Git
+
+```sh
+choco install git
+````
+
+### Install Meteor
+```sh
+choco install meteor
+```
+
+### Install ImageMagick (Optional but recommended)
+
+```sh
+choco install imagemagick
+```
+
+### Install Windows Build Tools 2015
+
+```sh
+npm install -g windows-build-tools
+```
+
+
+## Install Reaction
+
+### Install the Reaction command line client
+
+```sh
+# install CLI
+npm install -g reaction-cli
+```
+
+### Create your local Reaction installation
+
+You should preferably create a directory under your user, do not run this from the `\Windows\system32` directory as you won't have the correct permissions
+
+```sh
+# clone Reaction, install NPM dependencies
+reaction init
+cd reaction
+```
+
+If you don't want Reaction to be installed in the default `reaction` directory you can
+specify a direction like
+```sh
+reaction init my-new-reaction-project
+cd init my-new-reaction-project
+```
+
+
+### Start Reaction
+
+To start Reaction, run the `reaction` command
+
+```sh
+# start Reaction
+reaction
+# or
+reaction run
+```
```

Comment:
After `reaction run`, something like:

`Congrats! Now you've created your first Reaction store. View the store by going to http://localhost:3000 in your favorite browser.

To test your store as an admin user - go here. (link to store operator docs)
To start developing packages, go here. [package development documentation](/developer/packages/packages.md) 
To detailed instructions on modifying initial fixture data. [settings and import documentation](/developer/core/import.md) 
...
`